src/Symfony/Component/Notifier/Bridge/Octopush/OctopushTransport.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Octopush;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class OctopushTransport extends AbstractTransport
+{
+    protected const HOST = 'www.octopush-dm.com';
```

Comment:
```suggestion
    protected const HOST = 'octopush-dm.com';
```

Reply:
The issued certificate is valid only with the sub-domain 'www'.

Reply:
Ok maybe add this as a comment above the const, to avoid someone else will remove it?